userspace/falco/app_actions/configure_interesting_sets.cpp

```diff
@@ -15,15 +15,52 @@ limitations under the License.
 */
 
 #include "application.h"
+#include "configure_interesting_sets.h"
+#include <unordered_set>
+#include <sinsp.h>
+#include <sstream>
 
 using namespace falco::app;
 
-void application::configure_interesting_sets()
+extern sinsp_evttables g_infotables;
+
+std::string concat_evttypes_names(std::unordered_set<std::string> const names)
+{
+	std::stringstream ss;
+	std::copy(names.begin(), names.end(),
+	std::ostream_iterator<std::string>(ss, ", "));
+	std::string names_str = ss.str();
+	return names_str.substr(0, names_str.size() - 2);
+}
+
+application::run_result application::configure_interesting_sets()
 {
-	/// TODO: in the next future we need to change the interface of `enforce_simple_ppm_sc_set`
-	/// and `enforce_sinsp_state_tp` APIs, they shouldn't require an inspector to be called!
+
 	std::unique_ptr<sinsp> inspector(new sinsp());
 
+	/* Get all PPME events from the rules as idx codes. */
+	std::set<uint16_t> rule_events;
+	std::string source = falco_common::syscall_source;
+	m_state->engine->evttypes_for_ruleset(source, rule_events);
+	std::unordered_set<uint32_t> ppme_events_codes(rule_events.begin(), rule_events.end());
+
+	/* Translate PPME codes to event names. */
+	std::unordered_set<std::string> rules_evttypes_names = inspector->get_events_names(ppme_events_codes);
+
+	/* TODO discuss if exposing this as sinsp API. Translate event names to PPM syscall codes. */
```

Comment:
@FedeDP and @jasondellaluce what do you think re this? We are kind of mapping forth and back in general, but I think that's totally ok. Can be refactor in the future, right now would give precedence to achieving feature completeness asap.